volto/bootstrap.rst

```diff
@@ -0,0 +1,89 @@
+.. _bootstrap-label:
+
+=============================
+Bootstrapping A Volto Project
+=============================
+
+Installing Plone
+================
+
+On order to run Volto you need a backend. This can be either Plone or
+Guillotina. For this course we will use Plone, you can download Plone at
+:file:`https://plone.org/download`.
+
+Installing dependencies
+=======================
+
+First step is to install the correct Node version using :file:`nvm`:
```

Comment:
Is `:file:` the proper directive to use throughout the docs? AFAIK, it's not a docutils or Sphinx directive. Perhaps it is custom? Perhaps this should be `:term:` and a Glossary file would be better? See above comment regarding Glossary.